test/unit/core/reducers/is-fetching.ts

```diff
@@ -0,0 +1,57 @@
+import { Actions, Store } from '../../../../src/core';
+import isFetching from '../../../../src/core/reducers/is-fetching';
+import suite from '../../_suite';
+
+suite('isFetching', ({ expect }) => {
+  const state: Store.IsFetching = {
+    moreRefinements: false,
+    moreProducts: false,
+    search: false,
+    autocompleteSuggestions: false,
+    autocompleteProducts: false,
+    details: false,
+  };
+
+  function expectStartFetching(type: string, section: string) {
+    it(`it should set ${section} to true on ${type}`, () => {
+      const reducer = isFetching(state, <any>{ type });
+
+      expect(reducer).to.eql({ ...state, [section]: true });
+    });
+  }
+
+  function expectDoneFetching(type: string, section: string) {
+    it(`it should set ${section} to false on ${type}`, () => {
+      const reducer = isFetching({ ...state, [section]: true }, <any>{ type });
```

Comment:
This value is not a reducer, but a reduced value. Perhaps `reduced` or `newState` is a better name?